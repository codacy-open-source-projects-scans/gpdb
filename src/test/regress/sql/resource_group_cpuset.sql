--
-- Test: cpuset cannot be specified when group is disabled.
--

-- start_ignore
DROP RESOURCE GROUP resource_group1;
-- end_ignore

CREATE RESOURCE GROUP resource_group1 WITH (cpuset='0');
CREATE RESOURCE GROUP resource_group1 WITH (cpu_max_percent=5);
ALTER RESOURCE GROUP resource_group1 SET cpuset '0';

DROP RESOURCE GROUP resource_group1;
