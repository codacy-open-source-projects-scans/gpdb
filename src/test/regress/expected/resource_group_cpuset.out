--
-- Test: cpuset cannot be specified when group is disabled.
--
-- start_ignore
DROP RESOURCE GROUP resource_group1;
ERROR:  resource group "resource_group1" does not exist
-- end_ignore
CREATE RESOURCE GROUP resource_group1 WITH (cpuset='0');
ERROR:  resource group must be enabled to use cpuset feature
CREATE RESOURCE GROUP resource_group1 WITH (cpu_max_percent=5);
WARNING:  resource group is disabled
HINT:  To enable set gp_resource_manager=group
ALTER RESOURCE GROUP resource_group1 SET cpuset '0';
ERROR:  resource group must be enabled to use cpuset feature
DROP RESOURCE GROUP resource_group1;
